website/src/types/ProblemDetail.ts

```diff
@@ -2,11 +2,11 @@ import { z } from 'zod';
 
 export const problemDetailSchema = z
     .object({
-        type: z.string().url().optional(),
+        type: z.string().optional(),
         title: z.string().optional(),
         status: z.number().int().optional(),
         detail: z.string().optional(),
-        instance: z.string().url().optional(),
+        instance: z.string().optional(),
```

Comment:
From the docs it seems that instance and the type should be an uri. Why did you change it here?

Reply:
It wasn't a URL in fact. 

Also this should be as resilient as possible. Here (on the website), it doesn't matter what kind of string it is. But it should not fail to parse objects, because then our error handling fails (which is bad!).